install.php

```diff
@@ -197,8 +197,8 @@ function print_form($d){
         <fieldset>
             <p><?php echo $lang['i_license']?></p>
             <?php
-            array_push($license,array('name' => $lang['i_license_none'], 'url'=>''));
-            if(empty($d['license'])) $d['license'] = 'cc-by-sa';
+            $license['none'] = array('name' => $lang['i_license_none'], 'url'=>'');
```

Comment:
`none` refers to an undefined value in configuration manager. `''` should be better and works on my end.

```suggestion
            $license[''] = array('name' => $lang['i_license_none'], 'url'=>'');
```